src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -77,6 +82,96 @@ def get_category_for(unit: Optional[str]) -> Optional[str]:
         return UnitDatabase.GetSingleton().GetDefaultCategory(unit)
 
 
+def update_multi_input_flags(
+    alfacase_content: DescriptionDocument, item_description: T
+) -> T:
+    """
+    Update the multi input flags in `item_description` if not present in `alfacase_content` and
+    can be unambiguously deduced from the presence of contant and transient inputs.
+
+    :returns: The updated `item_description` (can return `item_description` unchanged.
+    """
+    item_dict = attr.asdict(item_description, recurse=False)
+    fields_to_update: Dict[str, constants.MultiInputType] = {}
+    for key, value in item_dict.items():
+        if (
+            key.endswith(constants.MULTI_INPUT_TYPE_SUFFIX)
```

Comment:
That function will make the code removing suffixes really better.

About the check.
It is probably enough since that `Enum` has just this purpose, and any same input for this function will abide to that.
I added the extra check to guard against possible errors.
I could remove the suffix check altogether or move it to the if's body as `assert key.endswith(constants.MULTI_INPUT_TYPE_SUFFIX)` for safekeeping. (I am doing that extra suffix check somewhere else I think).

I will move it to the if's body. If you have any objection let me know.

Reply:
> I added the extra check to guard against possible errors.

But this will actually hide errors, like someone using a different suffix but as a `MultiInput` value... that's the main point of my question, just checking the instance is not enough?

Reply:
I did remove the suffix check from the condition.
But let the check as asserts. If in the future we try to repurpose the Enum we will have a clear indication of code that need to be updated.